CHANGES.md

```diff
@@ -99,8 +99,8 @@ Release Notes.
 * Expose more ElasticSearch health check related logs to help to
   diagnose `Health check fails. reason: No healthy endpoint`.
 * Add source `event` generated metrics to SERVICE_CATALOG_NAME catalog.
-* [Breaking Change] Deprecate `All` from OAL source.
-* [Breaking Change] Remove `SRC_ALL: 'All'` from OAL grammar tree.
+* [**Break Change**] Deprecate `All` from OAL source.
```

Comment:
@wu-sheng, out of curiosity why change to "Break Change" from "Breaking Change"? The latter made more sense.

Reply:
I was trying to change in another way